docs/source/llms.txt.template

```diff
@@ -0,0 +1,186 @@
+# SkyPilot
+
+SkyPilot is a system for running, managing, and scaling AI workloads on any AI infrastructure.
+
+SkyPilot gives AI teams a simple interface to run jobs on any infra. Infra teams get a unified control plane to manage any AI compute — with advanced scheduling, scaling, and orchestration.
```

Comment:
Maybe we can include the home page content here, so that AI can understand the value adds of SkyPilot better? https://docs.skypilot.co/en/latest/docs/index.html